bluesky/suspenders.py

```diff
@@ -162,13 +163,10 @@ def get_futures(self):
         -------
         futs : list
             List of futures to wait on
-
-        msg : str
-            String explaining why the suspender is tripped
         '''
         if not self.tripped:
-            return [], ''
-        return [self.__make_event().wait()], self.get_justification()
```

Comment:
Here the API is reverted to what it was. I gather that this was a quick fix and not an especially intentional API choice. But feel free to argue why we shouldn't just call `get_justification()` separately, @tacaswell.